docs/intro.rst

```diff
@@ -268,6 +263,8 @@ environment (i.e. without using a UNIX-like environment; for that, see
 
       > pip install pywbem
 
+.. _`Chocolatey package manager`: https://chocolatey.org/
```

Comment:
Should we say more than just Chocolatey package manager and a url???

Reply:
This is just the definition of a link target; it does not generate anything in the text. It is used a few lines up.